scalding-core/src/main/scala/com/twitter/scalding/TypedDelimited.scala

```diff
@@ -47,15 +48,15 @@ trait TypedSeperatedFile extends Serializable {
 /**
  * Typed tab separated values file
  */
-object TypedTsv extends TypedSeperatedFile {
```

Comment:
we could take the fields passed in, require that they have the same size as the fields from the TypedDescriptor, then make a new TypeDescriptor that takes just the names from the input fields (but copies the types out).